contracts/wrapper/NameWrapper.sol

```diff
@@ -102,6 +103,54 @@ contract NameWrapper is
         return super.ownerOf(id);
     }
 
+    function safeTransferFrom(
```

Comment:
What made overriding this necessary?

Reply:
This was for a bug that allowed .eth names to be transferred during grace period